bundles/org.openhab.binding.souliss/src/main/java/org/openhab/binding/souliss/SoulissBindingConstants.java

```diff
@@ -0,0 +1,218 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.souliss;
+
+import java.util.Collections;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link SoulissBinding} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Tonino Fazio - Initial contribution
+ * @author Luca Calcaterra - Refactor for OH3
+ */
+@NonNullByDefault
+public final class SoulissBindingConstants {
+
+    public static final String BINDING_ID = "souliss";
+
+    public static final long DISCOVERY_RESEND_TIMEOUT_IN_MILLIS = 5000;
+    public static final int DISCOVERY_TIMEOUT_IN_SECONDS = 60;
+    public static final long SERVER_CICLE_IN_MILLIS = 10;
+    public static final long SEND_DISPATCHER_MIN_DELAY_CYCLE_IN_MILLIS = 10;
+
+    // public static final int PING_resendTimeoutInSeconds = 5;
```

Comment:
Please remove any code that is commented out.

Reply:
I removed All unnecessary comments, but mantaining some useful code in case of future implementations.